bdsbilling/src/main/java/com/appcoins/wallet/bdsbilling/repository/RemoteRepository.kt

```diff
@@ -22,8 +24,23 @@ class RemoteRepository(private val api: BdsApi, private val responseMapper: BdsA
   }
 
   internal fun getSkuDetails(packageName: String, skus: List<String>): Single<List<Product>> {
-    return api.getPackages(packageName, skus.joinToString(separator = ","))
-        .map { responseMapper.map(it) }
+    return requestSkusDetails(packageName, skus).map { responseMapper.map(it) }
+  }
+
+  private fun requestSkusDetails(packageName: String,
+                                 skus: List<String>): Single<DetailsResponseBody> {
+    return if (skus.size <= SKUS_DETAILS_REQUEST_LIMIT) {
+      api.getPackages(packageName,
+          skus.take(SKUS_DETAILS_REQUEST_LIMIT).joinToString(separator = ","))
```

Comment:
Is this take needed? 
At this point the sku list will have less or 50 items . so take the first 50 it's redundant right?